EventsExpress/ClientApp/src/actions/logout.js

```diff
@@ -4,17 +4,25 @@ import { reset_hub } from './chat';
 import { resetNotification } from './chats';
 export const SET_LOGOUT = "SET_LOGOUT";
 
-  export default function logout(){
-    localStorage.clear();
-    return dispatch => { 
+export default  function logout() {
+       RevokeToken();
```

Comment:
The method is not awaited. Consider awaiting async methods.

Reply:
deleted async/await